### PR TITLE
BXMSDOC-4443 7.5 BA release notes

### PR DESCRIPTION
**Pull request by emmurphy1**
Details in jira:
[BXMSDOC-4443](https://issues.jboss.org/browse/BXMSDOC-4443)

Also, many resolved comments in [PR 206](https://github.com/emmurphy1/kie-docs/pull/206).

### CODE REVIEW

File: doc-content/enterprise-only/installation/patches-applying-proc.adoc [hunk 1]
```diff
@@ -109,15 +109,19 @@ The update script creates a `backup` folder in the extracted `{PRODUCT_INIT}-$VE
...
+. If you use the standalone *{PRODUCT} {PRODUCT_VERSION_LONG} Maven Repository* artifact (`{PRODUCT_FILE}-maven-repository.zip`), such as in air-gap environments, download *{PRODUCT} {PRODUCT_VERSION}.x Maven Repository* and extract the downloaded `{PRODUCT_INIT}-{PRODUCT_VERSION}.x-maven-repository.zip` file to your existing `~/maven-repository` directory to update the relevant contents.
 +
 --
 Example Maven repository update:
 [source,subs="attributes+"]
 ----
-$ unzip -o {PRODUCT_INIT}-{PRODUCT_VERSION}.1-incremental-maven-repository.zip -d $REPO_PATH/{PRODUCT_INIT}-{PRODUCT_VERSION}.0-maven-repository/maven-repository/
+$ unzip -o {PRODUCT_INIT}-{PRODUCT_VERSION}.1-maven-repository.zip 'rhba-{PRODUCT_VERSION}.1.GA-maven-repository/maven-repository/*' -d /tmp/rhbaMavenRepoUpdate
+
+$ mv /tmp/rhbaMavenRepoUpdate/rhba-{PRODUCT_VERSION}.1.GA-maven-repository/maven-repository/ $REPO_PATH/{PRODUCT_INIT}-{PRODUCT_VERSION}.0.GA-maven-repository/maven-repository/
```
Comment (winklerm) [inline doc-content/enterprise-only/installation/patches-applying-proc.adoc]: This is the suggestion I meant in the chat:
```suggestion
$ mv /tmp/rhbaMavenRepoUpdate/rhba-{PRODUCT_VERSION}.1.GA-maven-repository/maven-repository/ $REPO_PATH/
```
Provided the $REPO_PATH is not defined anywhere else or does not have a special meaning defined elsewhere, I think this way could be simpler and users might get it. $REPO_PATH would be just the directory of their local Maven repository, depending on them which directory they use.

But maybe it was here for a reason, in that way, feel free to keep it as it is.

**Review by winklerm — APPROVED** [review]
The incremental Maven repository related changes look fine to me, thanks!

File: doc-content/enterprise-only/release-notes/rn-750-fixed-issues-ref.adoc [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+ifdef::DM[]
+== Decision engine
+* An abbreviated comparison with Map fails in the executable model	[https://issues.jboss.org/browse/RHDM-1034[RHDM-1034]]
+* A `BigDecimal` and primitive comparison error occurs in the executable model build	[https://issues.jboss.org/browse/RHDM-1039[RHDM-1039]]
+* When the `from accumulate` statement uses `BigDecimal` with the `MVEL` syntax, it fails with the executable model	[https://issues.jboss.org/browse/RHDM-1033[RHDM-1033]]
+* A `ClassCastException` error occurs in the `MVEL` dialect when updating a `list` property with `modify`	[https://issues.jboss.org/browse/RHDM-1062[RHDM-1062]]
+* A comparison error occurs with primitive and Map in the executable model	[https://issues.jboss.org/browse/RHDM-1035[RHDM-1035]]
+* A compilation error occurs for the `from accumulate init()` statement	[https://issues.jboss.org/browse/RHDM-1032[RHDM-1032]]
+* An error occurs while using Declared Types in different packages	[https://issues.jboss.org/browse/RHDM-1037[RHDM-1037]]
```
Comment (michelehaglund) [inline doc-content/enterprise-only/release-notes/rn-750-fixed-issues-ref.adoc]: Should "Declared Types" be tagged?

File: doc-content/enterprise-only/release-notes/rn-750-fixed-issues-ref.adoc [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+* A `NullPointerException` occurs at `org.drools.core.phreak.SegmentUtilities.createChildSegments` with a removed rule in `updateToVersion`	[https://issues.jboss.org/browse/RHDM-1066[RHDM-1066]]
+* Salience assignment behavior is different when `SequentialMaxPriority` is set	[https://issues.jboss.org/browse/RHDM-1031[RHDM-1031]]
+* Type coercion occurs with `Map.get()`` after jitting	[https://issues.jboss.org/browse/RHDM-1022[RHDM-1022]]
+* Using `BigDecimal` with `from accumulate` causes an `Error: argument type mismatch` error	[https://issues.jboss.org/browse/RHDM-1027[RHDM-1027]]
+* KJAR dependencies are not correctly set in the `kie-maven-plugin` build	[https://issues.jboss.org/browse/RHDM-1058[RHDM-1058]]
+* The executable model fails with a query with more than five arguments	[https://issues.jboss.org/browse/RHDM-1011[RHDM-1011]]
+* The `kie-maven-plugin` does not generate the `kbase.cache` in the KJAR [https://issues.jboss.org/browse/RHDM-1001[RHDM-1001]]
+* Line separators in the *CONDITION* column in the spreadsheet are escaped	[https://issues.jboss.org/browse/RHDM-1029[RHDM-1029]]
```
Comment (michelehaglund) [inline doc-content/enterprise-only/release-notes/rn-750-fixed-issues-ref.adoc:74]: "are escaped" What fresh hell is that?

File: doc-content/enterprise-only/release-notes/rn-750-fixed-issues-ref.adoc [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+//* Edited boolean columns in guided decision tables do not support aliases with objects	[https://issues.jboss.org/browse/RHDM-982[RHDM-982]]
+* An empty structure causes the DMN editor to crash	[https://issues.jboss.org/browse/RHDM-1076[RHDM-1076]]
+* {CENTRAL} fails to create a social event file on Windows when the user login contains special characters [https://issues.jboss.org/browse/RHDM-813[RHDM-813]]
+* The main class (`ClientProducerDemo`) of the High Available Event-Driven Decisioning sample project is not able to start because of a missing configuration	[https://issues.jboss.org/browse/RHDM-1081[RHDM-1081]]
+* Rule-based test scenarios that contain maps of simple types fail	[https://issues.jboss.org/browse/RHDM-1015[RHDM-1015]]
+
+
+== Decision Model and Notation (DMN)
+* When you import a PMML model into a DMN model and then save or validate the model, DMN throws an error	[https://issues.jboss.org/browse/RHDM-1077[RHDM-1077]]
+* The included models feature does not work correctly when you include a BKM node from other DMN model. 	[https://issues.jboss.org/browse/RHDM-978[RHDM-978]]
```
Comment (michelehaglund) [inline doc-content/enterprise-only/release-notes/rn-750-fixed-issues-ref.adoc]: Isn't a period at the end of the sentence inconsistent?

**Review by michelehaglund — APPROVED** [review]
Minor inline comments. Looks good.

**Pull request closed without merging**